spec/base_model_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "spec_helper"
+require "cfa/augeas_parser"
+require "cfa/base_model"
+require "cfa/memory_file"
+
+describe CFA::BaseModel do
+  let(:parser) { CFA::AugeasParser.new("sudoers.lns") }
+  let(:path) { "/etc/sudoers" }
+  subject { CFA::BaseModel.new(parser, path)}
+
+  describe ".default_file_handler" do
+    it "returns object set by .default_file_handler" do
```

Comment:
NP: I'd say this block is redundant to the last one. I know it's looking to the same thing from both the setter and the getter, but a little bit too much for my taste. :-)